### ISSUE TITLE
replace with latest buildAddrOutput

### ISSUE THREAD

**Issue by xhliu** [sCrypt-Inc/tic-tac-toe#24]
https://github.com/sCrypt-Inc/tic-tac-toe/blob/3d6b4d06ae752a22b15d4c2b6415586302542e9b/src/contracts/tictactoe.ts#L52-L56

**Comment by ankana2113** [comment]
I want to work on this as a contributor to TSoC '24. Can you please assign the issue to me?

**Comment by xhliu** [comment]
> I want to work on this as a contributor to TSoC '24. Can you please assign the issue to me?

Of course. Thanks for helping with this. Feel free to take other pending issues as well.
